docs/markdown/Tutorials/create-a-new-goal.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Create a new goal"
 slug: "create-a-new-goal"
-excerpt: "Getting started writing plugins for Pants by creating a new goal."
+excerpt: "Getting started writing plugins for Pants by creating a new goal"
```

Comment:
Removing the dot to make consistent with the rest of the tutorials.

Reply:
To me a well formed sentence ends with a period. Maybe add the missing period on the other excerpts instead?